SharkCage/CageService/CageService.cpp

```diff
@@ -35,7 +35,7 @@ std::optional<HANDLE> CageService::CreateImpersonatingUserToken()
 		return std::nullopt;
 	}
 
-	if (!tokenLib::aquireTokenWithPrivilegesForTokenManipulation(appropriate_token))
+	if (!tokenLib::aquireTokenWithPrivilegesForTokenManipulation(appropriate_token)) 
```

Comment:
Pls remove the white space 😅.

Reply:
Have you been thinking about creating some explicit coding standards? Because honestly... I am starting to be a little confused by what exactly do you guys care for regarding your code style.

Reply:
The only points we explicitely talked about was naming of variables and classes. Everything else is basically the Format-Tools of VS2017 (`ctrl+k, ctrl+d`). If anything is not covered by these two points we just like to have it consistent with the rest of the existing code. Is there anything in particular you find _confusing_?